thyrel-web/src/types/HolyElement.type.ts

```diff
@@ -8,6 +8,7 @@ export type HolyElement = {
   finishAt: string;
   createdAt: string;
   drawImage: string;
+  initiatorId: number;
```

Comment:
```suggestion
  initiatorId?: number;
```

attention il est pas tjr la